collection/collection/build.gradle

```diff
@@ -64,7 +66,7 @@ kotlin {
         commonMain {
             dependencies {
                 api(libs.kotlinStdlib)
-                implementation("org.jetbrains.compose.annotation-internal:annotation:1.7.1")
+                implementation(project(":annotation:annotation"))
```

Comment:
Aren't we going to depend on annotation published by androidx since it's available for all targets? 
Why do we need a project dependency? 

Although I think we'll stop publishing collections too and depend on collections published by androidx, right?

Reply:
> Aren't we going to depend on annotation published by androidx since it's available for all targets?

We should, but separately. It's not a part of merging/updating

> Why do we need a project dependency?

Initially it's because of merged, but it seems valid - https://android-review.googlesource.com/c/platform/frameworks/support/+/3329155 updates it to a latest version

> Although I think we'll stop publishing collections too and depend on collections published by androidx, right?

Yes. It's a good description for the separate ticket. Not a part of merging process too.

Reply:
Added to the scope of CMP-7183